modules/side-panel/react/lib/SidePanel.tsx

```diff
@@ -196,16 +187,17 @@ export default class SidePanel extends React.Component<SidePanelProps, SidePanel
 
   public render() {
     const {
+      backgroundColor = SidePanelBackgroundColor.White,
+      openNavigationLabel = 'open navigation',
+      closeNavigationLabel = 'close navigation',
+      openDirection = SidePanelOpenDirection.Left,
```

Comment:
This default won't be picked up in the private `toggleButtonDirection` function below.